Tutorials/whatiswot/1.WoT/1.WoT.vtt

```diff
@@ -0,0 +1,110 @@
+WEBVTT
+
+00:00:00.000 --> 00:00:04.889 
+[pause 5.5s]
+
+00:00:04.889 --> 00:00:10.293 
+Hello! Welcome to the "What is WoT" video series. [pause strong]
```

Comment:
this `[]` is leaking in the captions subtitles. I am not sure if it is intended... but they distract a little bit.

Reply:
@idilsezgin there is an option to exclude the pauses in murf. Could you reupload them with that?

Reply:
.vtt files are generated again without the brackets.